src/test/java/uk/gov/hmcts/reform/civil/utils/DocmosisTemplateDataUtilsTest.java

```diff
@@ -15,6 +15,23 @@
 
 class DocmosisTemplateDataUtilsTest {
 
+    @Test
+    void shouldReturnNull_whenBothApplicant1NotFount() {
+        CaseData caseData = CaseData.builder()
+            .respondent1(Party.builder()
+                             .type(Party.Type.INDIVIDUAL)
+                             .individualTitle("Mr.")
+                             .individualFirstName("Alex")
+                             .individualLastName("Richards")
+                             .partyName("Mr. Alex Richards")
+                             .build())
+            .ccdCaseReference(1L)
+            .build();
+        String caseName = toCaseName.apply(caseData);
+        assertThat(caseData.getApplicant1()).isNull();
```

Comment:
we are asserting the field is null which was never set to case data. This does not add  any value to unit tests. 

I would expect to pass  a caseData to `fetchApplicantName` method where you made changes, and assert the returned value or exception (if expected to be thrown) etc.

Reply:
done